examples/basics/quick_start.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>Suggestion: highlight library names with <code>code</code> :  <code>labelbox</code> , <code>labelbox[data]</code>   and <code>uuid</code>  (lower case for the library name)</p><p>For curious people: add "universal" before unique (to explain the extra "u" in <code>uuid</code> ) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done!